plugins/@grouparoo/sendgrid/package.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Please note a few change are needed to `package.json` to match the master branch:

* changes to `engines`
* changes to versions of @grouparoo packages in the monorepo
* actionhero, fs-extra, typescript and others have new versions